cmd/web/routes.go

```diff
@@ -8,12 +8,22 @@ import (
 	"net/http"
 )
 
+type IRoutes interface {
+	Hello(c *gin.Context)
+	Ping(c *gin.Context)
+}
+
 type WebHandler struct {
 	Logger *zap.Logger
 	R      *gin.Engine
 	db     *gorm.DB
 }
 
+func invokeRoutes(r *gin.Engine, h IRoutes) {
```

Comment:
I've moved these here as I don't like going anywhere other than here to look for the routes contract. Having it on the di file seems to make sense at first but I prefer it here.